front/src/features/books/booksSlice.tsx

```diff
@@ -70,32 +85,55 @@ export const booksSlice = createSlice({
           )
         : [...state.basket, item];
     },
+    cleanBasket: (state) => {
+      state.basket = [];
+    },
   },
 });
 
 export const {
   fetchBooksDataStarted,
   fetchBooksDataSucceeded,
   fetchBooksDataFailed,
+  sendFormStarted,
+  sendFormSucceeded,
+  sendFormFailed,
   addItemToBasket,
   incrementValue,
   decrementValue,
   deleteItem,
   setValue,
+  setOrderCompleted,
+  setOrderNotCompleted,
+  cleanBasket,
 } = booksSlice.actions;
 
+export const sendForm = (data: object) => async (dispatch: any) => {
```

Comment:
nie uzywaj typu 'object', znajdz jaki to jest dokladnie typ. 
zamiast dispatch: any wystarczy samo dispatch, u mnie ts to ogarnia. A any uzywaj tylko w ostatecznosci, jak cos nie dziala i jest dziwne

Reply:
Done